common/cuda_api_wrappers/CMakeLists.txt

```
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.17)
```

@wep21 
Thank you for the suggestion. It looks a cool alternative.
According to [README](https://github.com/eyalroz/cuda-api-wrappers#requirements), it seems to require cmake version 3.18 or later. So how about to set the required version to `cmake_minimum_required` ?
```suggestion
cmake_minimum_required(VERSION 3.18)
```